src/NuGet.Core/NuGet.PackageManagement/Projects/ProjectItemProperties.cs

```diff
@@ -9,11 +9,12 @@ namespace NuGet.ProjectManagement
     /// </summary>
     public static class ProjectItemProperties
     {
-        public const string IncludeAssets = "IncludeAssets";
-        public const string ExcludeAssets = "ExcludeAssets";
-        public const string PrivateAssets = "PrivateAssets";
+        public const string IncludeAssets = nameof(IncludeAssets);
+        public const string ExcludeAssets = nameof(ExcludeAssets);
+        public const string PrivateAssets = nameof(PrivateAssets);
```

Comment:
I don't support these changes. Customers use the strings IncludeAssets, ExcludeAssets and PrivateAssets in their project files, so we must support those exact strings forever. If someone renames the field name, the value should not change.

Reply:
We won't change these strings, but we already follow this pattern a lot. 
It eliminates any misspelling hick-ups.

Reply:
As discussed directly, I disagree that this does anything to eliminate misspellings, and I feel like it's a misuse of a language feature.